cmd/dockerd/daemon.go

```diff
@@ -431,6 +431,12 @@ func loadDaemonCliConfig(opts *daemonOptions) (*config.Config, error) {
 		return nil, err
 	}
 
+	if opts.Validate {
```

Comment:
Wondering why this code was put on this position, and not further below (given that more options are validated below)

Reply:
Moved to bottom of the block.